### ISSUE TITLE
RIS import ignores L2 ("Full-text link") field

### ISSUE THREAD

**Issue by buhtz** [JabRef/jabref#2607]
Using JabRef 3.8.1 on Debian unstable.

I exported entries from Zotero as RIS format including the external files (e.g. snapshots). Zotero create a RIS-file and a `file`-directory for the external files.

e.g.
```

TY  - ELEC
TI  - git - Pull new updates from original GitHub repository into forked GitHub repository - Stack Overflow
UR  - http://stackoverflow.com/questions/3903817/pull-new-updates-from-original-github-repository-into-forked-github-repository
Y2  - 2016/08/30/11:53:28
L2  - files/5888/pull-new-updates-from-original-github-repository-into-forked-github-repository.html
```

When importing that into JabRef it looks like that the `L2` field is totaly ignored.
JabRef should be aware of that field and move that file to the setup "General file directory".

The same problem is when using "Biblatex" as export format in Zotero.

Currently this makes JabRef unusable for me because I am not able to import my data from Zotero. No snapshots of websites not PDF files etc

**Comment by buhtz** [comment]
When looking into the code of `RisImproter.java` it looks like that JabRef even doesn't support the real RIS. See #2609.

**Comment by matthiasgeiger** [comment]
Bibtex/biblatex export works fine for me...
All linked files/snapshots are accessible from within JabRef.

**Comment by tokkot** [comment]
I found what I think is a more up-to-date specification for the RIS format in the manual for Reference Manager 12, [here](https://researchsoftware.com/sites/researchsoftware.com/files/product_attachments/v12_refman_userguide_0.pdf), page 581. But Reference Manager has been discontinued in favor of EndNote, so many EndNote defines the spec now.

**Comment by lenhard** [comment]
This should be fixed by #3642
